test.sh

```diff
@@ -0,0 +1 @@
+PA11Y_PATH=/usr/local/bin/pa11y docker-compose run scan gsa.gov --scan=a11y
```

Comment:
Kind of sort of ... This _could_ be the beginnings of some simple testing, but I can remove it if that makes things easier.